lua/core/clock.lua

```diff
@@ -233,9 +233,13 @@ function clock.add_params()
       norns.state.clock.link_quantum = x
     end)
   params:set_save("link_quantum", false)
-
+  local clock_table = {"off"}
+  for i = 2,17 do
```

Comment:
maybe more clear to do `i=1,16` and then below, `clock_table[i+1]` and then get rid of all the `-1`

Reply:
lol, YES, _absolutely_

Reply:
@dndrks this was the thing I had the question about, rad 👍🏻